frontend/hub/repositories/RepositoryPage.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  DateTimeCell,
+  LoadingPage,
+  PageActions,
+  PageHeader,
+  PageLayout,
+  useGetPageUrl,
+} from '../../../framework';
+import { HubRoute } from '../HubRoutes';
+import { PageRoutedTabs } from '../../../framework/PageTabs/PageRoutedTabs';
+import { useParams } from 'react-router-dom';
+import { useGet } from '../../common/crud/useGet';
+import { Repository } from './Repository';
+import { pulpAPI } from '../api/formatPath';
+import { DropdownPosition } from '@patternfly/react-core/deprecated';
+import { useRepositoryActions } from './hooks/useRepositoryActions';
+import { PulpItemsResponse } from '../usePulpView';
+import { parsePulpIDFromURL } from '../api/utils';
+import { StatusCell } from '../../common/StatusCell';
+import { Trans, useTranslation } from 'react-i18next';
+import { HubError } from '../common/HubError';
+
+export function RepositoryPage() {
+  const params = useParams<{ id: string }>();
+  const headerActions = useRepositoryActions();
+  const getPageUrl = useGetPageUrl();
+  const { t } = useTranslation();
+
+  const { data, error, refresh } = useGet<PulpItemsResponse<Repository>>(
+    params.id ? pulpAPI`/repositories/ansible/ansible/?name=${params.id}` : ''
+  );
+  if (error) return <HubError error={error} handleRefresh={refresh} />;
+  if (!data) return <LoadingPage breadcrumbs tabs />;
+
+  const breadcrumbs = [
+    { label: t('Repositories'), to: getPageUrl(HubRoute.Repositories) },
+    { label: params.id },
+  ];
+
+  const repository = data?.results.length > 0 ? data?.results[0] : undefined;
+  const repo_id: string = parsePulpIDFromURL(repository ? repository.pulp_href : '') || '';
+  return (
+    <>
+      <PageLayout>
+        <PageHeader
+          title={params.id}
+          headerActions={
+            <PageActions
+              actions={headerActions}
+              position={DropdownPosition.right}
+              selectedItem={repository}
+            />
+          }
+          footer={
+            !!repository &&
+            !!repository.last_sync_task && (
+              <div>
+                <Trans>
+                  Last updated from registry{' '}
+                  {<DateTimeCell value={repository.last_sync_task?.finished_at} />}
+                </Trans>
+                <StatusCell status={repository.last_sync_task?.state} />
+              </div>
+            )
+          }
+          breadcrumbs={breadcrumbs}
+        />
+        <PageRoutedTabs
+          tabs={[
+            {
+              label: 'Details',
```

Comment:
Do we need to select those labels for translation?

Reply:
I missed that. Fixed.